moveit_core/robot_state/include/moveit/robot_state/cartesian_interpolator.h

```diff
@@ -144,6 +152,75 @@ class CartesianInterpolator
     double meters;
   };
 
+  /** \brief Compute the sequence of joint values that correspond to a straight Cartesian path for a particular link.
+
+     The Cartesian path to be followed is specified as a \e translation vector to be followed by the robot \e link.
+     This vector is assumed to be specified either in the global reference frame or in the local
+     reference frame of the link.
+     The resulting joint values are stored in the vector \e traj, one by one. The interpolation distance in
+     Cartesian space between consecutive points on the resulting path is specified in the \e MaxEEFStep struct which
+     provides two fields: translation and rotation. If a \e validCallback is specified, this is passed to the internal
+     call to setFromIK(). In case of IK failure, the computation of the path stops and the value returned corresponds to
+     the distance that was achieved and for which corresponding states were added to the path.
+
+     The struct CartesianPrecision specifies the precision to which the path should follow the Cartesian straight line.
+     If the deviation at the mid point of two consecutive waypoints is larger than the specified precision, another waypoint
+     will be inserted at that mid point. The precision is specified separately for translation and rotation.
+     The maximal resolution to consider (in percentage of the total path length) is specified by max_resolution.
```

Comment:
So is it percentage or a fraction? Given its default value (`1e-5`) I assume fraction?

Reply:
If you mean that fraction and percentage are different by a factor of 100, you are right: a fraction (values between 0 and 1) is meant here.

Reply:
Exactly what I meant!